docs/common-parameters.md

```diff
@@ -7,6 +7,8 @@ Parameters shared by all goals.
 | installDirectory | Local installation directory of the Liberty server. | Yes, only when `runtimeArchive`, `runtimeArtifact`, and `install` parameters are not set. |
 | runtimeArchive | Location of the Liberty server compressed archive. The archive will be unpacked into a directory as specified by the `runtimeInstallDirectory` parameter. | Yes, only when `installDirectory`, `runtimeArtifact`, and `install` parameters are not set. |
 | runtimeArtifact | Maven artifact name of the Liberty runtime. The runtime will be installed into a directory as specified by the `runtimeInstallDirectory` parameter. The default runtime is the latest version of `io.openliberty:openliberty-kernel`. | No |
+| libertyRuntimeGroupId | Liberty runtime groupId to use instead of the `runtimeArtifact` groupId. This can also be specified with `-Dliberty.runtime.groupId` from the command line. | No |
+| libertyRuntimeArtifactId | Liberty runtime artifactId to use instead of the `runtimeArtifact` artifactId. This can also be specified with `-Dliberty.runtime.artifactId` from the command line. | No |
```

Comment:
What about the new behavior where you are checking that it is not empty, and if it is that results in the default being used? I think something needs to be added about that. Plus there may be another doc that talks about the overall behavior of specifying `libertyRuntimeVersion` that could use updating (or an example of usage).

Reply:
Maybe `installation-configuration.md` could be updated to mention these properties and the overriding behavior (and the behavior when specifying an empty string)?